gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/diagnosticreport/SpecimenCompoundsMapperTest.java

```diff
@@ -159,6 +169,53 @@ public void testHandlingNonUserNarrativeStatement() {
         assertThat(observation.getIssuedElement().asStringValue()).isNull();
     }
 
+    @Test public void testOrderingIsPreservedForDiagnosticReportResults() {
+        final RCMRMT030101UKEhrExtract  ehrExtract =
+            unmarshallEhrExtract("specimen_with_three_test_group_headers.xml");
```

Comment:
```suggestion
        final RCMRMT030101UKEhrExtract ehrExtract =
            unmarshallEhrExtract("specimen_with_three_test_group_headers.xml");
```

Another little thing, two spaces! 😆